frontend/src/metabase/routes.jsx

```diff
@@ -241,7 +241,13 @@ export const getRoutes = store => (
           <Route path=":slug/:objectId" component={QueryBuilder} />
         </Route>
 
-        <Route path="/model/:slug/detail" component={ModelDetailPage} />
+        <Route path="/model/:slug/detail">
+          <IndexRoute component={ModelDetailPage} />
```

Comment:
It's possible to enter the model detail without having any tab selected. I don't think that's intended, since on stats if you go to the model detail page the **Used by** tab will be open by default. Maybe it's use `<IndexRedirect />` instead?

```suggestion
          <IndexRedirect to="usage" />
```